scripts/lua2dox.lua

```diff
@@ -315,6 +315,34 @@ function TLua2DoX_filter.filter(this, AppStamp, Filename)
     local line
     local fn_magic -- function name/def from  magic comment
 
+    outStream:writeln([=[
+
+
+#include <assert.h>
+
+#include "klib/kvec.h"
+
+#ifdef INCLUDE_GENERATED_DECLARATIONS
+# include "api/buffer.c.generated.h"
+#endif
+
+/// @defgroup lua-treesitter-languagetree
```

Comment:
Doesn't work :( Also tried `\defgroup`.

But it works in api/buffer.c ... https://github.com/neovim/neovim/blob/ef52592cf8bc158a91b276017d1d2e68a620e7e0/src/nvim/api/buffer.c#L47-L49